src/components/languageMenu/LanguageMenuPure.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import OutsideClickHandler from 'react-outside-click-handler'
+import LanguageContext from 'src/contexts/language'
+import './language-menu.less'
+import configPages from 'src/config'
+
+const LanguageMenuPure = ({
+  closeLanguageMenu,
+  toggleLanguageMenu,
+  isLanguageMenuShown
+}) => {
+  const locationPathname = window.location.href
+    .split('/')
+    .filter(function(element) {
+      return element !== ''
+    })
+    .pop()
+  if (
+    configPages[locationPathname] &&
+    configPages[locationPathname].languages !== -1
+  ) {
+    const currentLanguage = LanguageContext._currentValue
+    const currentAvailableLanguage = configPages[locationPathname].languages
+    const domainHref = 'https://platinum.fund/'
+
+    const currentLanguageContent = (
+      <button
+        className="languages-menu__current-option"
+        onClick={toggleLanguageMenu}
+      >
+        <img
+          alt="Language en"
+          className="languages-menu__option-image"
+          src={require('./images/languages/' + currentLanguage + '.png')}
+          data-option="en"
+        />
+        <p className="languages-menu__option-name">{currentLanguage}</p>
+      </button>
+    )
+
+    const currentLanguageList = currentAvailableLanguage.map(
```

Comment:
Could you please move it outside of the function, i.e. make it as a React component. And probably move to another file in the same directory